src/themes/uswitch/theme.json

```diff
@@ -2385,9 +2385,15 @@
         },
         "linkBlock": {
           "boxShadow": "0px 0px 8px rgba(0, 0, 0, 0.45)",
-          "width": ["220px", "296px", "296px"],
+          "maxWidth": ["220px", "296px", "296px"],
```

Comment:
are max widths necessary? feel like with width we just let it grow to whatever container its in

Reply:
looks really ugly though, when container is very wide

Reply:
but that depends on usage actually, list items could potentially also be wide - i'll remove min widhs